Gemfile.lock

```diff
@@ -121,16 +118,6 @@ GEM
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    mailcatcher (0.2.4)
```

Comment:
why is mailcatcher being removed?

Reply:
It doesn't work as a standalone binary when included in the Gemfile. In this case, it's probably a stale branch though.